extensions/wikia/WikiaRSS/WikiaRssExternalController.class.php

```diff
@@ -32,24 +34,54 @@ public function getRssFeeds() {
 				return;
 			}
 
-			$status = null;
-			$rss = @fetch_rss($url, $status);
+			$feedData = WikiaDataAccess::cacheWithLock(
+				wfSharedMemcKey( 'WikiaRss', 'feed', $url ),
```

Comment:
I don't like using method name for cache key. I don't see a reason to expire cache when we move a method into other place.